packs/aws/sensors/sqs_sensor.py

```diff
@@ -0,0 +1,95 @@
+"""
+This is generic SQS Sensor using boto3 api to fetch messages from sqs queue.
+After receiving a messge it's content is passed as payload to a trigger 'aws.sqs_new_message'
+
+This sensor can be configured either by using config.yaml withing a pack or by creating
+following values in datastore:
+    - aws.input_queue
```

Comment:
For consistency with other sensor, it would be good to nest those options under `sqs_sensor`.

So something like that:

```yaml
service_notifications_sensor:
  host: "localhost"
  port: 12345
  path: "/my-path"

sqs_sensor:
  input_queue: "fooo"
```